src/include/catalog/catalog_sql_table.h

```diff
@@ -27,7 +27,7 @@ class SqlTableHelper {
    * Constructor
    * @param table_oid the table oid of the underlying sql table
    */
-  explicit SqlTableHelper(catalog::table_oid_t table_oid) : table_oid_(table_oid) {}
```

Comment:
Students are going to misuse this class. We should rename it to `CatalogSqlTableHelper`

Reply:
I plan on completely eliminating this class in the refactor.  I think it is obfuscating large portions of the catalog code rather than making it easier to read and reason about.